src/Apps/Shows/Routes/ShowsCity.tsx

```diff
@@ -60,14 +60,10 @@ export const ShowsCity: React.FC<ShowsCityProps> = ({
 
   const [loading, setLoading] = useState(false)
 
-  const currentShowsRef = useRef<HTMLDivElement | null>(null)
-  const { scrollTo } = useScrollToElement({
-    selectorOrRef: currentShowsRef,
-    offset: 20,
-  })
+  const { jumpTo } = useJump({ offset: 20 })
 
   const handleClick = (cursor: string, page: number) => {
-    scrollTo()
+    jumpTo("CurrentShows")
```

Comment:
**question**: maybe it would be better if we keep `id` as constant?

Reply:
Updated in this file; though I don't really feel strongly either way. I suppose it could help to export these to keep them in sync across multiple components. Though now that it's an `id` on a purpose built component, you know now what it does and it's less likely to be removed or changed because of that.